wagtail/wagtailadmin/edit_handlers.py

```diff
@@ -695,10 +696,15 @@ def __init__(self, relation_name, panels=None, label='', help_text='', min_num=N
         self.max_num = max_num
 
     def bind_to_model(self, model):
+        if django.VERSION >= (1, 9):
```

Comment:
Is it worth adding this to `wagtail.utils.compat`? There are already two similar methods in there from the changes between 1.7/1.8.